subt/ros/robot/src/filter_pointcloud.py

```diff
@@ -77,6 +77,7 @@ def points_callback(self, msg):
                               row_step=cloud_struct.size * msg.width,
                               data=new_data)
         self.points_publisher.publish(new_msg)
+        self.debug_data = new_data
```

Comment:
this is dirty way how to get the raw data to the test ... it could be probably extract it also from MagicMock of PointCloud2